drivers/regulator/axp803.c

```diff
@@ -515,6 +509,8 @@ axp803_probe(struct device *dev)
 	if ((err = regulator_set_defaults(dev, (uint16_t *)dev->drvdata)))
 		return err;
 
+	dev->subdev_count = AXP803_REGL_COUNT;
```

Comment:
Should this not be in the probe function? The probe function will for sure be called before this, so there should be no reason to have t his here.

Reply:
It *is* in the probe function.

Reply:
My bad. I was looking at it condensed on github